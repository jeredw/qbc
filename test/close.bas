close 1, #2, 3
close 1
close
