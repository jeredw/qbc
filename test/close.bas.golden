(program 
  (statement 
    (close_statement close 
      (expr 
        (literal 1)) , # 
      (expr 
        (literal 2)) , 
      (expr 
        (literal 3)))) \n 
  (statement 
    (close_statement close 
      (expr 
        (literal 1)))) \n 
  (statement 
    (close_statement close)) \n)
